src/legacy/ui/public/agg_types/utils.tsx

```diff
@@ -17,6 +17,15 @@
  * under the License.
  */
 
```

Comment:
This file doesn't look like it needs to be a `.tsx` file, could it be a `.ts` file?

Reply:
Sure. Updated.